includes/admin/class-wc-rest-payments-terminal-locations-controller.php

```diff
@@ -45,17 +45,31 @@ public function register_routes() {
 	 * @return WP_REST_Response|WP_Error
 	 */
 	public function get_store_location( $request ) {
-		try {
-			$name    = get_bloginfo();
-			$address = [
-				'city'        => WC()->countries->get_base_city(),
-				'country'     => WC()->countries->get_base_country(),
-				'line1'       => WC()->countries->get_base_address(),
-				'line2'       => WC()->countries->get_base_address_2(),
-				'postal_code' => WC()->countries->get_base_postcode(),
-				'state'       => WC()->countries->get_base_state(),
-			];
+		$name             = get_bloginfo();
+		$store_address    = WC()->countries;
+		$location_address = [
+			'city'        => $store_address->get_base_city(),
+			'country'     => $store_address->get_base_country(),
+			'line1'       => $store_address->get_base_address(),
+			'line2'       => $store_address->get_base_address_2(),
+			'postal_code' => $store_address->get_base_postcode(),
+			'state'       => $store_address->get_base_state(),
+		];
+
+		// If address is not populated, emit an error and specify the URL where this can be done.
+		$provided_address       = array_filter( $location_address );
+		$is_address_unpopulated = 2 === count( $provided_address ) && isset( $provided_address['country'], $provided_address['state'] );
+		if ( $is_address_unpopulated ) {
+			return rest_ensure_response(
+				new \WP_Error(
+					'store_address_is_incomplete',
+					__( 'The store address is incomplete, please actualize settings.', 'woocommerce-payments' ),
+					[ 'url' => admin_url( add_query_arg( [ 'page' => 'wc-settings' ], 'admin.php' ) ) ]
```

Comment:
Should it also contain `&tab=general` here? E.g.  it's `wp-admin/admin.php?page=wc-settings&tab=general` in my case

Reply:
It's the default tab, but I added the tab parameter in 26bacd1a096856f3e55d91cddf5fa4f077a28165